eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -5617,27 +5617,6 @@ public function testDeleteTranslationFromDraftThrowsInvalidArgumentException()
         $contentService->deleteTranslationFromDraft($draft->versionInfo, $languageCode);
     }
 
-    /**
-     * Simplify creating custom role with limited set of policies.
-     *
-     * @param $roleName
-     * @param array $policies e.g. [ ['content', 'create'], ['content', 'edit'], ]
-     */
-    private function createRoleWithPolicies($roleName, array $policies)
```

Comment:
@andrerom merge commit 70a2706 (`6.13` into `7.0`) didn't include this removal done in https://github.com/ezsystems/ezpublish-kernel/commit/be17f3de0ff7a9b6d464df8d1568a446ec1e301a#diff-896f1fe3af332cf2a9e38b343ebe9367L5643 for `6.12`, so adding this here as well. 
I'm not sure why this happened, though :)

Reply:
Conflict with removal of TranslationInfo stuff, so lets just remove it here and be done with it.